libcxx/test/std/utilities/format/format.arguments/format.arg/visit.pass.cpp

```diff
@@ -10,18 +10,26 @@
 
 // <format>
 
+// class basic_format_arg;
+
+// template<class Visitor>
+//   decltype(auto) visit(this basic_format_arg arg, Visitor&& vis);  // since C++26
```

Comment:
Please create a new test for the non-member version. Feel free to copy paste the original test.

Reply:
Just to confirm: You propose to move the new test cases for member `visit` into a new file and restore the original contents, is that correct?
For `variant` and `format` I added the new test cases into the already existing test files, similar to other tests.

Reply:
yes, the current change removes testing for C++20 and C++23.